.github/workflows/cd-workflow.yml

```
@@ -0,0 +1,137 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: ☕️ set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+          server-id: github
+          setting-path: ${{ github.workspace }}
```

actionlint (1.7.7)

input "setting-path" is not defined in action "actions/setup-java@v4". available inputs are "architecture", "cache", "cache-dependency-path", "check-latest", "distribution", "gpg-passphrase", "gpg-private-key", "java-package", "java-version", "java-version-file", "jdkFile", "job-status", "mvn-toolchain-id", "mvn-toolchain-vendor", "overwrite-settings", "server-id", "server-password", "server-username", "settings-path", "token"

(action)